c2corg_images/__init__.py

```diff
@@ -2,18 +2,25 @@
 from pyramid.renderers import JSON
 from pyramid.events import NewRequest
 import os
+import json
 
-RESIZING_CONFIG = [
-    {'suffix': 'BI', 'convert': ['-resize', '1500x1500>',
-                                 '-quality', '90']},
-    {'suffix': 'MI', 'convert': ['-resize', '400x400>',
-                                 '-quality', '90']},
-    {'suffix': 'SI', 'convert': ['-resize', '200x200^',
-                                 '-gravity', 'center',
-                                 '-extent', '200x200',
-                                 '-quality', '90']}
-]
+def parse_resizing_config(string):
```

Comment:
I considered `resizing_config` but `string` is more explicit about the format of the config.